.github/CONTRIBUTING.md

```diff
@@ -448,22 +448,22 @@ for(var/atom/thing as anything in bag_of_atoms)
 	* Areas should not be var-edited on a map to change it's name or attributes. All areas of a single type and it's altered instances are considered the same area within the code, and editing their variables on a map can lead to issues with powernets and event subsystems which are difficult to debug.
 
 ### User Interfaces
-* All new player-facing user interfaces must use TGUI.
+* All new player-facing user interfaces should use TGUI.
```

Comment:
This could be said to depend on one's opinion of TGUI, again needs a justification before watering down code standards